system/lib/libc/musl/arch/emscripten/bits/syscall.h

```diff
@@ -113,6 +112,21 @@
 #define __NR_recvmmsg		337
 #define __NR_prlimit64		340
 #define __NR_sendmmsg		345
+#define __NR_socket		359
+#define __NR_socketpair		360
+#define __NR_bind		361
+#define __NR_connect		362
+#define __NR_listen		363
+#define __NR_accept4		364
+#define __NR_getsockopt		365
+#define __NR_setsockopt		366
+#define __NR_getsockname		367
+#define __NR_getpeername		368
+#define __NR_sendto		369
+#define __NR_sendmsg		370
+#define __NR_recvfrom		371
+#define __NR_recvmsg		372
+#define __NR_shutdown		373
```

Comment:
Does musl establish this numbering, or do we? Will this stay in sync with musl?

Reply:
Oh nm, reading https://github.com/emscripten-core/musl/commit/c2feda4e2ea61f4da73f2f38b2be5e327a7d1a91 suggests these are well established.

It looks like this was a recent change to musl upstream, did we update musl already before to support this? (or do we not really need to do so to update?)

Reply:
IIUC the recent change to upstream musl was to avoid socketcall multiplexing on more architecutres, but I think the idea was that any architectures that support the split out syscalls shoudl already have been using them.  (i.e. it was arguably a bug that they were using socketcall).